index.js

```diff
@@ -942,7 +977,7 @@
   //. > S.init([])
   //. Nothing()
   //. ```
-  S.init = R.ifElse(R.isEmpty, Nothing, R.compose(Just, R.init));
+  S.init = S.slice(0, -1);
```

Comment:
> This also allows us to define `S.at` in terms on `S.slice` too.

I don't see it, but I'll wait to see your implementation. :)

The important thing is that `S.at(-0, xs)` should return a Nothing for every `xs`, since in this context `-0` refers to the position *after* the last index.